pkg/apis/policy/types.go

```diff
@@ -63,6 +63,7 @@ type PodDisruptionBudgetStatus struct {
 	// the list automatically by PodDisruptionBudget controller after some time.
 	// If everything goes smooth this map should be empty for the most of the time.
 	// Large number of entries in the map may indicate problems with pod deletions.
+	// +optional
```

Comment:
@liggitt is this comment useful for internal types as well?

Reply:
I can't think of anything that gets generated off the internal types, so I don't think so